dam/loans/templates/loans/loanItem.html

```diff
@@ -37,6 +37,10 @@ <h2 class="card-title" style="color: #ffffff;">Client Details</h2>
                         <p>Reserved on: </p>
                         <p>{{ reserved.reserved_at }}</p>
                         <hr>
+                        <br>
+                        <p>Due on: </p>
+                        <p>{{ reserved.due_on }}</p>
```

Comment:
```suggestion
                        <p>{{ reserved.item.loan_duration.days }}</p>
```
This would show how many days the item can be loaned for so the manager has some info.